dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -970,3 +970,13 @@ CWWKD1097.param.incompat=CWWKD1097E: The {0} method of the {1} repository \
 CWWKD1097.param.incompat.explanation=Order, Sort, and Limit parameters are \
  not compatible with the type of repository operation.
 CWWKD1097.param.incompat.useraction=Remove the Order, Sort, or Limit parameter.
+
+CWWKD1098.spec.param.position.err=CWWKD1098E: The {0} method of the {1} repository \
+ has a {2} parameter that is positioned before the query parameters. \
+ Parameter types that have special meaning in Jakarta Data ({3}) must be \
+ positioned after the other parameters in the repository method signature.
+CWWKD1098.spec.param.position.err.explanation=When special Jakarta Data defined \
+ parameter types are included in a repository method signture, they must be \
+ positioned after all of the query parameters.
```

Comment:
Change "all of the" to "all the" or "all" (remove "of").

I use "of" when I talk, but the "of" gets flagged by a content quality checker.

Reply:
Thanks for pointing that out. I added commit https://github.com/OpenLiberty/open-liberty/pull/30405/commits/2208ef554f19120a714f75fefaeb220a1ea23ed4 to correct it.